src/unity/python/turicreate/toolkits/image_similarity/image_similarity.py

```diff
@@ -409,3 +409,92 @@ def similarity_graph(self, k=5, radius=None, include_self_edges=False,
         +----------+----------+----------------+------+
         """
         return self.similarity_model.similarity_graph(k, radius, include_self_edges, output_type, verbose)
+
+    def export_coreml(self, filename, reference_data):
+        """
+        Save the model in Core ML format.
+
```

Comment:
Is client code needed to sort the output and pick the top k? If so, it's probably worth a mention here.